.golangci.yml

```diff
@@ -43,5 +43,5 @@ linters:
     - stylecheck
     - wsl
     - interfacer
+    - gomnd
```

Comment:
Disabling the magic number detector, it just leads to overly verbose code. We should use our own discretion.